### PR TITLE
Add basic generated structure

### PR DESCRIPTION
**Pull request by jowczar**
This PR adds basic generated structure (constants, routes, images, etc.) from DhiWise.

### CODE REVIEW

File: frontend/lib/core/utils/size_utils.dart [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+import 'package:flutter/material.dart';
+
+// This is where the magic happens.
+// This functions are responsible to make UI responsive across all the mobile devices.
+
+Size size = WidgetsBinding.instance.window.physicalSize /
+    WidgetsBinding.instance.window.devicePixelRatio;
+
+// Caution! If you think these are static values and are used to build a static UI,  you mustn’t.
```
Comment (ShanghaiG) [inline frontend/lib/core/utils/size_utils.dart:11]: Love this comments, if theyre autogenerated, its hell of a feature from this dhiWise

File: frontend/lib/screens/common_widgets/text_field.dart [hunk 2]
```diff
@@ -171,7 +171,7 @@ class _CustomTextFieldState extends State<CustomTextField> {
   //       image: AssetImage(
```
Comment (ShanghaiG) [inline frontend/lib/screens/common_widgets/text_field.dart:171]: hmmm, why is this commented out :d

File: frontend/lib/screens/lp_unauthorized_screen/lp_unauthorized_screen.dart [hunk 3]
```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/material.dart';
+import 'package:workout_app/core/app_export.dart';
+
+class LpUnauthorizedScreen extends StatelessWidget {
```
Comment (ShanghaiG) [inline frontend/lib/screens/lp_unauthorized_screen/lp_unauthorized_screen.dart:4]: LoginPageUnathorizedScreen? thats the abbv for Lp?

**Review by ShanghaiG — APPROVED** [review]

File: frontend/lib/screens/lp_unauthorized_screen/lp_unauthorized_screen.dart (hunk 3 above)
Comment (jowczar) [reply frontend/lib/screens/lp_unauthorized_screen/lp_unauthorized_screen.dart:4]: landing page – page that appears after splash

**Pull request merged**